src/features/docs/view/Stoplight.tsx

```diff
@@ -27,7 +25,21 @@ const Stoplight = ({ url }: { url: string }) => {
   )
 }
 
-const ResponsiveStoplight = ({ document }: { document: string }) => {
+const ResponsiveStoplight = ({ document: apiDescriptionDocument }: { document: string }) => {
+  useEffect(() => {
+    const script = document.createElement("script")
+    script.src = "https://unpkg.com/@stoplight/elements/web-components.min.js"
```

Comment:
We should probably lock the version of elements.

Reply:
Good point! That's addressed in https://github.com/shapehq/framna-docs/pull/489/commits/d2afe2b517d2c351b34edca8677139874dbdce1a.